staging/src/k8s.io/apiextensions-apiserver/examples/client-go/apis/cr/v1/doc.go

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// +k8s:deepcopy-gen=package
```

Comment:
Yes, it's just moved code plus fixups. Here it seems I left in the deepcopy tag in leading to the generated code (which is not used yet, but needed in the static copy pr).